openedx/core/djangoapps/notifications/utils.py

```diff
@@ -47,6 +48,19 @@ def get_show_notifications_tray(user):
     return show_notifications_tray
 
 
+def get_is_new_notification_view_enabled():
+    """
+    Returns True if the waffle flag for the new notification view is enabled, False otherwise.
+    """
+    flag_model = get_waffle_flag_model()
+
+    try:
+        flag = flag_model.objects.get(name=ENABLE_NEW_NOTIFICATION_VIEW.name)
+        return flag.everyone if flag.everyone is not None else False
+    except flag_model.DoesNotExist:
+        return False
```

Comment:
To check if `WaffleFlag` is enabled, use `.is_enabled()` because no course or user information is required. Custom implementation will be helpful when waffle flag needs to be enabled for a specific user/s. 


```
def get_is_new_notification_view_enabled():
    """
    Returns True if the waffle flag for the new notification view is enabled, False otherwise.
    """
    return ENABLE_NEW_NOTIFICATION_VIEW.is_enabled()
```